packages/admin/src/Charcoal/Admin/Template/System/ClearCacheTemplate.php

```diff
@@ -55,6 +58,20 @@ class ClearCacheTemplate extends AdminTemplate
      */
     private $apcCacheKeyPattern;
 
+    /**
+     * Mustache View Engine.
+     *
+     * @var \Charcoal\View\Mustache\MustacheEngine
+     */
+    private $mustacheEngine;
+
+    /**
+     * Twig View Engine.
+     *
+     * @var \Charcoal\View\Twig\TwigEngine
+     */
+    private $twigEngine;
```

Comment:
Either type these properties against `MustacheEngine` and `TwigEngine` or both against `EngineInterface`.